### PR TITLE
Misc bugfixes

### PR DESCRIPTION
**Pull request by troglobit**
## Description

Misc. fixes to annoying regressions and missing functionality.

## Checklist

Tick *relevant* boxes, this PR is-a or has-a:

- [X] Bugfix
  - [ ] Regression tests
  - [x] ChangeLog updates (for next release)
- [ ] Feature
  - [ ] YANG model change => revision updated?
  - [ ] Regression tests added?
  - [ ] ChangeLog updates (for next release)
  - [ ] Documentation added?
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (please detail in commit messages)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: src/statd/python/yanger/yanger.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def insert(obj, *path_and_value):
     curr[path[-1]] = value
 
 
-def run_cmd(cmd, testfile, default=None):
+def run_cmd(cmd, testfile, default=None, quiet=False):
```
Comment (wkz) [inline src/statd/python/yanger/yanger.py:105]: For people familiar with Python's `subprocess.run()`, I think `check` is more familiar.

```suggestion
def run_cmd(cmd, testfile, default=None, check=True):
```

Then we can change `s/subprocess.check_output(...)/subprocess.run(..., check=check).output` below.

That way, a non-zero exitcode won't raise an exception, so there's no need for the `if not quiet` guard below.

File: doc/container.md [hunk 2]
```diff
@@ -222,9 +222,9 @@ archive, which helps greatly with container upgrades (see below):
 Upgrading a Container Image
 ---------------------------
 
-> **Note:** the default writable layer is lost when upgrading the image
-> Use named volumes for directories with writable content you wish to
-> keep over an upgrade.
+> **Note:** the default writable layer is lost when upgrading the image.
+> Use named volumes for directories with writable content that you wish
+> to keep over an upgrade.
```
Comment (wkz) [inline doc/container.md:227]:
```suggestion
> Use named volumes for directories with writable content that you wish
> to keep over an upgrade.
```
```suggestion
> Use named volumes for content that you want to persist across upgrades.
```

Something about "keep over an upgrade" doesn't feel quite right, but this is just my opinion.

**Review by wkz — APPROVED** [review]
Great work tracking this down! :heart: 

Just found a couple of stylistic things, do with them what you will.

File: src/statd/python/yanger/yanger.py (hunk 1 above)
Comment (troglobit) [reply src/statd/python/yanger/yanger.py:105]: Ah, of course, great idea thanks!

**Pull request merged**